### ISSUE TITLE
Changes in custom CSS are ignored on incremental documentation rebuild

### ISSUE THREAD

**Issue by atyrin** [Kotlin/dokka#2286]
For a configured project with a custom CSS (and probably assets), incremental documentation rebuild ignores changes in static files.

Dokka configured for multiModule project. The configuration contains:
```Kotlin
pluginConfiguration<org.jetbrains.dokka.base.DokkaBase, org.jetbrains.dokka.base.DokkaBaseConfiguration> {
//            customAssets = listOf(file("<path to asset>"))
        customStyleSheets = listOf(file("custom.css"))
    }
```
I perform changes in `custom.css` file and re-run task `dokkaHtmlMultiModule`. But Gradle marks the task as `UP-TO-DATE`.
`build clean` / `--rerun-tasks` / changes in `build.gradle.kts` helps

**Installation**
- Dokka version: 1.6.10

**Comment by solarday** [comment]
The same behavior in case you make any changes in templates.

**Comment by whyoleg** [comment]
Fixed in [Dokka 2.0.0](https://github.com/Kotlin/dokka/releases/tag/v2.0.0) in [Dokka Gradle plugin v2﻿](https://kotlinlang.org/docs/dokka-migration.html) by providing type safe configuration with Gradle types for `html` plugin
